com.unity.render-pipelines.high-definition/Editor/RenderPipeline/HDRenderPipelineMenuItems.cs

```diff
@@ -93,6 +93,75 @@ static void CreateSceneSettingsGameObject(MenuCommand menuCommand)
             volume.sharedProfile = profile;
         }
 
+        [MenuItem("Edit/Render Pipeline/HD Render Pipeline/Force Forward Emissive on Material/Enable in Selection")]
+        internal static void ForceForwardEmissiveOnMaterialEnableInSelection()
+        {
+            var selection = UnityEditor.Selection.objects;
+
+            foreach (var obj in selection)
+            {
+                if (obj is Material)
+                {
+                    Material material = obj as Material;
```

Comment:
```suggestion 
                if (obj is Material material)
                {
```

If you put the name `material` just after the `is Material` it declares the variable, so you don't have to create and cast it afterwards.

Reply:
thanks